### ISSUE TITLE
Can't update project

### ISSUE THREAD

**Issue by Marenz** [frequenz-floss/frequenz-repo-config-python#99]
### What happened?

While trying to update [my actor project](https://github.com/Marenz/frequenz-actor-fcr-balancing/tree/sdk-update), I got this error:

```
❰marenz❙~/frequenz/fcr(git≠sdk-update)❱✘≻ cookiecutter gh:frequenz-floss/frequenz-repo-config-python --directory=cookiecutter  --replay --replay-file .cookiecutter-replay.json --overwrite-if-exists
You've downloaded /home/marenz/.cookiecutters/frequenz-repo-config-python before. Is it okay to delete and re-download it? [yes]:
Traceback (most recent call last):
  File "/usr/bin/cookiecutter", line 33, in <module>
    sys.exit(load_entry_point('cookiecutter==2.1.1', 'console_scripts', 'cookiecutter')())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/cookiecutter/cli.py", line 194, in main
    cookiecutter(
  File "/usr/lib/python3.11/site-packages/cookiecutter/main.py", line 114, in cookiecutter
    result = generate_files(
             ^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/cookiecutter/generate.py", line 373, in generate_files
    generate_file(
  File "/usr/lib/python3.11/site-packages/cookiecutter/generate.py", line 173, in generate_file
    tmpl = env.get_template(infile_fwd_slashes)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/jinja2/environment.py", line 1010, in get_template
    return self._load_template(name, globals)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/jinja2/environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/jinja2/loaders.py", line 138, in load
    code = environment.compile(source, name, filename)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/jinja2/environment.py", line 768, in compile
    self.handle_exception(source=source_hint)
  File "/usr/lib/python3.11/site-packages/jinja2/environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File ".github/labeler.yml", line 13, in template
    - "**/*.md"
  ^^^^^^^^^^^^^^
jinja2.exceptions.TemplateAssertionError: No filter named 'as_identifier'.
  File ".github/labeler.yml", line 13
    #   - "src/frequenz/{{cookiecutter.type}}/{{cookiecutter.name | as_identifier}}/module/**"
```

### What did you expect instead?

no error.

### Affected part(s)

Cookiecutter template (part:cookiecutter)

**Comment by Marenz** [comment]
Nevermind, reading release notes helps. But maybe it could be mentioned in the upgrade docu for this specific release

**Comment by llucax** [comment]
It is in the release notes for this specific release: https://github.com/frequenz-floss/frequenz-repo-config-python/releases/tag/v0.4.0

Do you mean somewhere else? BTW, I'll close as invalid.

**Comment by Marenz** [comment]
I meant [the upgrade guide](https://github.com/frequenz-floss/frequenz-repo-config-python/blob/v0.x.x/docs/index.md#update-an-existing-project) could have had this specific note mentioned, but the mention should only exist in this particular version `v0.4.0` and would be removed again in `v0.5.0` from that guide. 
Or possibly replaced with new notes that are valid only for `v0.5.0`
That would be my ideal scenario I suppose :)

Basically, the release notes contain the chronological history of things to watch out for when upgrading, but the upgrade guide contains the note only important to upgrade from the previous to the current (and maybe a hint to check the release notes for other upgrade scenarios)

**Comment by llucax** [comment]
> I meant [the upgrade guide](https://github.com/frequenz-floss/frequenz-repo-config-python/blob/v0.x.x/docs/index.md?rgh-link-date=2023-08-02T10%3A43%3A26Z#update-an-existing-project) could have had this specific note mentioned, but the mention should only exist in this particular version v0.4.0 and would be removed again in v0.5.0 from that guide.
> Or possibly replaced with new notes that are valid only for v0.5.0
> That would be my ideal scenario I suppose :)

I see and agree that's much more convenient from the user's perspective, but it is quite annoying from the maintainers perspective.

What we could do instead is add a generic note to the ~migration~ upgrade instructions in the docs that mentions that you should check the **Upgrading** section on the release notes for the version(s) you want to upgrade to. I guess that could have a better balance, because also when upgrading if you are skipping versions (which is totally valid) you'll never have all the list of steps necessary if we reset the migration instructions in the docs on every release.

**Comment by Marenz** [comment]
No, not the migration instructions, the upgrade instructions

**Comment by llucax** [comment]
Yeah, sorry, it is confusing that we have an upgrading section in the release notes and an upgrading instructions in the docs. I mean the upgrading instructions in the docs can mention the upgrading section in the release notes for the version you are upgrading to :)

**Comment by llucax** [comment]
So, would that be sufficient or at least better in you view?

**Comment by Marenz** [comment]
Def. better

**Comment by llucax** [comment]
* #103
